### PR TITLE
feat: Add smart-contract example

### PR DESCRIPTION
**Pull request by yashrajdesai**
### Description
This PR adds an example for executing smart contracts with the help of [Call Engine.](https://iroha.readthedocs.io/en/develop/develop/api/commands.html#call-engine) 

I have referred [this](https://github.com/hyperledger/iroha/blob/main/example/burrow_integration/get_transactions.py) code to add getAccountTransactions example using smart-contract.

### CODE REVIEW

File: example/smart-contract/getAccountTransactions.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
...103610100830152610f968184610cdc565b90509a9950505050505050505050565b60006020820190508181036000830152610fbf81610d15565b9050919050565b6000610fd0610fe1565b9050610fdc82826110d3565b919050565b6000604051905090565b600067ffffffffffffffff82111561100657611005611104565b5b61100f82611147565b9050602081019050919050565b600081519050919050565b600081519050919050565b600082825260208201905092915050565b600081905092915050565b600082825260208201905092915050565b600061106a82611071565b9050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b82818337600083830152505050565b60005b838110156110be5780820151818401526020810190506110a3565b838111156110cd576000848401525b50505050565b6110dc82611147565b810181811067ffffffffffffffff821117156110fb576110fa611104565b5b80604052505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b600080fd5b600080fd5b600080fd5b600080fd5b6000601f19601f8301169050919050565b7f4572726f722063616c6c696e67207365727669636520636f6e7472616374206660008201527f756e6374696f6e0000000000000000000000000000000000000000000000000060208201525056fea26469706673582212205e584d5874226b894bcbcd685c22e9010a188385180559c526ec96dc806b313264736f6c63430008070033'
```
Comment (baziorek) [inline example/smart-contract/getAccountTransactions.ts:42]: Could You provide what is the code?

**Review by baziorek — APPROVED** [review]
For me it looks OK, just correct my single suggestion.
Unfortunately I'm not javascript programmer.

**Review by appetrosyan — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "ed25519.js": "^1.3.0",
     "google-protobuf": "^3.9.1",
     "js-sha3": "^0.8.0",
+    "keccak": "^3.0.2",
```
Comment (stefashkaa) [inline package.json:42]: if it's required only for examples and `examples` is in `.npmignore` - it's better to put it into `devDependencies`

Comment (yashrajdesai) [reply package.json]: Yes, made the changes accordingly.

**Review by appetrosyan — APPROVED** [review]

**Pull request merged**